Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs

```diff
@@ -68,12 +68,14 @@ static MixedRealityEditorSettings()
                     message += "- Force Text Serialization\n";
                 }
 
+#if UNITY_WSA
```

Comment:
Why are we wrapping this?
Does this fix something we were not aware of?

Reply:
Since we're on Android, you don't want IL2CPP to be enforced. Code debugging is only possible on a C++ basis then. I was not sure which way to comment this, I started with !UNITY_ANDROID but was not sure.

Reply:
Same for UWP. Please revert this change.

Reply:
I don't understand. If I remove it, IL2CPP will be enforced, which is not good on Android but fine with UWP. Or did I missunderstood something?

Reply:
It's not enforced. It asks you to update it. Although from what I understand IL2CPP for android is also the recommended setting as well, correct?

All of the settings suggested here are optional.

Reply:
>Although from what I understand IL2CPP for android is also the recommended setting as well, correct?

We prefer to compile to .NET on Android. We'll often prototype / test iOS apps on Android because .NET helps with rapid iteration. We actually made the same exact change on our end because we were tired of seeing the warning. Worth opening another PR.

Reply:
> We prefer to compile to .NET on Android.

It's mono, not .NET

![image](https://user-images.githubusercontent.com/13334553/53916500-2a52b680-4017-11e9-9e41-e320b9eef799.png)

Reply:
I agree with Railboy. It's mandatory for UWP, most likely the better option on standalone, but not recommended on Android and therefore not worth the popup.
I will go with !UNITY_ANDROID then, ok?

Reply:
https://unity3d.com/learn/tutorials/topics/best-practices/il2cpp-mono

> You should use IL2CPP to both develop and ship your Project. If iteration times end up being too slow using IL2CPP, switch temporarily to the Mono Scripting Backend during development to improve iteration speed.

I can see the benefit for faster iteration times, but even Unity suggests IL2CPP for Android as well.
Isn't there a "don't show me this again" button on the popup to make that dialog disappear forever? I thought I added that.

Reply:
I check it out

Reply:
Agreed, our and Unity's recommendation is to use IL2CPP for all builds.  It's fine using mono for dev / testing, but IL2CPP for release.

Hence the startup prompt is optional and you can choose to ignore it as a dev